ui/components/app/auto-detect-token/auto-detect-token-modal.tsx

```diff
@@ -55,7 +54,6 @@ function AutoDetectTokenModal({
         category: MetaMetricsEventCategory.Navigation,
         properties: {
           chain_id: chainId,
-          locale: getCurrentLocale,
```

Comment:
Hey @salimtb 👋  any idea why this was needed in the first place?

Reply:
Ohhhh, we're trying to set a value of a function instead of the locale itself.  So assuming we need the locale, we could do:

```
const locale = useSelector(getCurrentLocale)
```

Is this correct?

Reply:
Right that fixes it, i haven't seen any trackEvent passing the `locale`, so i removed, i guess i can put it back just in case and merge 🙏  thanks!

Reply:
I think selecting the locale here is fine, although it looks like the events may automatically get locale added.  May not end up being necessary.

https://github.com/MetaMask/metamask-extension/blob/92e6b945eb54fafba491d9a84ca95c2e5fbaf932/app/scripts/controllers/metametrics.js#L761